src/main.c

```diff
@@ -53,6 +53,7 @@ const char *usage =
     "\t\t\tn=2 Two-wire UART, Reset by DTR\n"
     "\t\t\tn=3 Single-wire UART, Reset by RTS\n"
     "\t\t\tn=4 Two-wire UART, Reset by RTS\n"
+    "\t\t\tn=5 Single-wire UART, Hardware Reset, reset state read by CTS (modified mode 1/3)\n"
```

Comment:
It's all "hardware reset."  I'd say "Manual reset button" instead.

Reply:
Will probably want to update this after seeing below about `communication_mode` being a bitmap.  `n=5` is as you describe, but `n=6` will be `Two-wire UART, Manual Reset`. (You get it for free when communication_mode is a bitmap.)